_applications/my-first-radanalytics-app.adoc

```diff
@@ -0,0 +1,113 @@
+= My First RADanalytics Application
+:page-link: my-first-radanalytics-app
+:page-weight: 0
+:page-labels: [Python, S2I]
+:page-layout: markdown
+:page-menu_template: menu_tutorial_application.html
+:page-menu_items:
+:page-description: In this tutorial you will learn how to create a source-to-image application for Apache Spark from the ground up. The source code is based on the upstream Pi calculator from the Apache Spark project examples with a slight twist, the addition of a web server to create an on-demand calculation microservice.
+
+[[introduction]]
+== Introduction
+
+In this tutorial you will build a microservice which will calculate an approximate value for https://en.wikipedia.org/wiki/Pi[Pi] when requested by HTTP. You will learn how to create applications which utilize Apache Spark and that are deployed directly from a source repository to OpenShift with a single command.
+
+This tutorial will walk you through every step to creating, deploying and operating an Apache Spark driver application on OpenShift. You should expect to spend 30 to 60 minutes on these exercises, depending on your familiarity.
```

Comment:
total nit, is "every step of" better, as in every step of the way?
I can also see "step to"

Reply:
i think you're right, "step of" sounds better